baby-liminal-extension/src/args.rs

```diff
@@ -20,6 +20,16 @@ pub struct StoreKeyArgs {
     pub key: Vec<u8>,
 }
```

Comment:
Do we have to even expose `StoreKey` in the extension? I don't see a case when this would be useful. It's `verify` that we want.

Reply:
discussed offline